dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-reports.controller.js

```diff
@@ -6,21 +6,19 @@ const {
 const getUtilisationReports = async (req, res) => {
   try {
     const { bankId } = req.params;
+    const { reportPeriod } = req.body;
```

Comment:
As this is a 'GET' request there shouldn't really be a request body. Also, is it correct that we want to search by the whole report period? If so then the query to the DB should also take into account the end dates.

Reply:
These values now come from `req.query` (just for clarity sake I have upgraded this file to typescript)